schedule/yam/agama_auto_patterns.yaml

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
could you please check where we are storing the profiles, I thought was under `/auto/`.
By the way we are trying to avoid `auto` in the name of the file, instead using `unattended'.

Reply:
This is the yaml schedule file in schedule/yam/ ,  the original file is agama_auto.yaml

Reply:
Read again my comment please.
our profiles are under /auto/ folder, please put it there.
Regarding the name of the file, we should use `unattended` instead of `auto`.